File: app-java/src/main/java/com/eress/closer/util/LogUtil.java
package com.eress.closer.util;

import android.content.Context;
import android.util.Log;
import android.widget.Toast;

public class LogUtil {
    public static void m(String message) {
        Log.d("closer", ""+ message);
    }

    public static void ts(Context context, String message) {
        Toast.makeText(context, message + "", Toast.LENGTH_SHORT).show();
    }

    public static void tl(Context context, String message) {
        Toast.makeText(context, message + "", Toast.LENGTH_LONG).show();
    }
}

File: app-kotlin/src/main/kotlin/com/eress/closer/util/LogUtil.kt
package com.eress.closer.util

import android.content.Context
import android.util.Log
import android.widget.Toast

object LogUtil {
    fun m(message: String) {
        Log.d("closer", "" + message)
    }

    fun ts(context: Context, message: String) {
        Toast.makeText(context, message + "", Toast.LENGTH_SHORT).show()
    }

    fun tl(context: Context, message: String) {
        Toast.makeText(context, message + "", Toast.LENGTH_LONG).show()
    }
}
